deploy/charts/openebs-monitoring/dashboards/cStor/Overview Dashboard.json

```diff
@@ -0,0 +1,1411 @@
+          "placement": "bottom"
+        },
+        "tooltipOptions": {
+          "mode": "single"
+        }
+      },
+      "pluginVersion": "7.5.5",
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "sum(rate(container_cpu_usage_seconds_total{namespace=\"$namespace\"}[1m]) * on (name,pod)group_left(cstor_pool,namespace) openebs_pool_status) * 100",
+          "instant": false,
+          "interval": "",
+          "intervalFactor": 1,
+          "legendFormat": "CPU usage",
+          "refId": "A"
+        }
+      ],
+      "timeFrom": null,
+      "timeShift": null,
+      "title": "CPU usage in namespace $namespace ",
+      "transparent": true,
+      "type": "timeseries"
+    },
+    {
+      "datasource": "Prometheus",
+      "fieldConfig": {
+        "defaults": {
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "semi-dark-red",
+                "value": 300
+              }
+            ]
+          },
+          "unit": "percent"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 12,
+        "w": 5,
+        "x": 19,
+        "y": 0
+      },
+      "id": 48,
+      "options": {
+        "colorMode": "background",
+        "graphMode": "area",
+        "justifyMode": "auto",
+        "orientation": "auto",
+        "reduceOptions": {
+          "calcs": [
+            "last"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "text": {},
+        "textMode": "auto"
+      },
+      "pluginVersion": "7.5.5",
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "sum(rate(container_cpu_usage_seconds_total{namespace=\"$namespace\"}[1m]) * on (name,pod)group_left(cstor_pool,namespace) openebs_pool_status) by(pod,namespace) * 100",
+          "hide": false,
+          "instant": false,
+          "interval": "",
+          "legendFormat": "CPU used by pod {{pod}}",
+          "refId": "A"
+        },
+        {
+          "exemplar": true,
+          "expr": "sum(rate(kube_pod_container_resource_requests_cpu_cores{namespace=\"$namespace\"}[1m]) * on (name,pod)group_left(namespace) openebs_volume_status) by(pod,namespace) * 0.33 * 100",
+          "hide": false,
+          "instant": false,
+          "interval": "",
+          "legendFormat": "CPU requests by pod {{pod}}",
+          "refId": "B"
+        },
+        {
+          "exemplar": true,
+          "expr": "sum(rate(kube_pod_container_resource_limits_cpu_cores{namespace=\"$namespace\"}[1m]) * on (name,pod)group_left(namespace) openebs_volume_status) by(pod,namespace) * 0.33 * 100",
+          "hide": false,
+          "instant": false,
+          "interval": "",
+          "legendFormat": "CPU limits by pod {{pod}}",
+          "refId": "C"
+        }
```

Comment:
@rajaSahil Does this expr look good to you. I am not sure as this CPU limits & requests are shown for volumes only and usage is shown for pools.

Reply:
As discussed, these will be removed.